02_sequencing/README.md

```diff
@@ -0,0 +1,32 @@
+#Animation File Format
+
+The animation files are CSV (comma separated value) spreadsheet files.
+
+The first row contains the column names which are used to identify the contents of the column.
+Every subsequent row contains the data for each frame.
+
+The header names are:
+
+`FRAME_ID` - This stores the index of the frame.
+This column should contain integers.
+Lowest values will be displayed first.
+This column is optional. If undefined the frames will be displayed in the order they are in the CSV file. 
+
+`FRAME_TIME` - The amount of time the frame will remain for.
```

Comment:
It might make more sense to specify this in milliseconds.

Reply:
I second the millisecond proposal

Reply:
Okay I will change this

Reply:
Just changed it